packages/manager/src/features/NodeBalancers/NodeBalancerCreate.tsx

```diff
@@ -412,7 +418,11 @@ const NodeBalancerCreate = () => {
   ].filter((item) => Boolean(item.title));
 
   if (nodeBalancerFields.region) {
-    summaryItems.unshift({ title: `$${price}/month` });
+    summaryItems.unshift({
+      title: `$${
+        renderMonthlyPriceToCorrectDecimalPlace(Number(price)) ?? UNKNOWN_PRICE
+      }/month`,
+    });
```

Comment:
I understand the intention here, but I'm not a fan of the extra complexity. Because the  NodeBalancer price is hard-coded I don't see why we'd have to handle an unknown price.

Also, I'm not sure if the fallback logic makes sense here. 


```suggestion
    summaryItems.unshift({
      title: `$${
        renderMonthlyPriceToCorrectDecimalPlace(Number(price))
      }/month`,
    });
```


Also, it looks like `renderMonthlyPriceToCorrectDecimalPlace` won't properly handle a service that is $0. (not that we're going to many free products)

Reply:
Yeah, you're right that we really wouldn't end up needing to handle displaying an unknown price in this case because although `price` can be undefined, we're only displaying price in the summary once there is a valid region.
(And other than that, the fallback logic was just unnecessary there with the render function, and I probably just wasn't paying attention when doing some clean up, and missed it.)

As for:
> looks like renderMonthlyPriceToCorrectDecimalPlace won't properly handle a service that is $0. (not that we're going to many free products)

In this case and in any other instances where we're using this in Cloud, I wouldn't expect these products to be free. Unless there's a likely situation that we'd encounter that, I think it's safer to leave as is so we don't risk inaccurately advertising free products.